src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/DeleteSavedRecordingCommandTest.java

```diff
@@ -107,6 +107,12 @@ void shouldNotValidateIncorrectArgc(int argc) {
         MatcherAssert.assertThat(e.getMessage(), Matchers.equalTo(errorMessage));
     }
 
+    @Test
+    void shouldValidateRenamedRecording() {
```

Comment:
Please make this a `@ParameterizedTest` and test various different numeric suffixes to be sure they all work.